app/views/catalog/mastheads/_government_documents.html.erb

```diff
@@ -0,0 +1,14 @@
+<div id="masthead-container">
+  <div id="masthead" class="government-documents-masthead">
+    <h1>Government documents</h1>
+    <div>
+      <p>Stanford is a depository library for state, federal, UN, and EU government documents.</p>
+      <p>
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
```

Comment:
Yeah, probably. The link to it on the home page if "off" by default so if we did push this to prod there would be no way to get here to see this text. 

@jvine do you have text in mind for this?  I could also just kill the `<p>` and add it later if needed (although it looks a little silly w/o it).